platform/core.ui/src/org/netbeans/core/ui/sysopen/SystemOpenAction.java

```diff
@@ -50,8 +50,10 @@
 @ActionID(id = "org.netbeans.core.ui.sysopen.SystemOpenAction", category = "Edit")
 @ActionRegistration(displayName = "#CTL_SystemOpenAction", lazy=false)
 @ActionReferences ({
+    @ActionReference(path = "Editors/TabActions", position = 401),
     @ActionReference(path = "UI/ToolActions/Files", position = 2045),
-    @ActionReference(path = "Projects/Actions", position = 101)
+    @ActionReference(path = "Projects/Actions", position = 101),
+    @ActionReference(path = "Shortcuts", name = "SO-S")
```

Comment:
SOS?

Reply:
Yes, hadn't realised that's what `ALT-SHIFT-S` would be until working on the PR. :smile:  Made sense before as similar to the terminal / copy path actions, with S for system (and O is taken).